docs/OV_Runtime_UG/Python_API_exclusives.md

```diff
@@ -1,93 +1,93 @@
+After the call to `wait_all`, jobs and their data can be safely accessed. Acquiring of a specific job with `[id]` returns `InferRequest` object, which results in seamless retrieval of the output data.
 
 @snippet docs/snippets/ov_python_exclusives.py asyncinferqueue_access
 
-#### Setting callbacks
+#### Setting Callbacks
 
-Another feature of `AsyncInferQueue` is ability of setting callbacks. When callback is set, any job that ends inference, calls upon Python function. Callback function must have two arguments. First is the request that calls the callback, it provides `InferRequest` API. Second one being called "userdata", provides possibility of passing runtime values, which can be of any Python type and later used inside callback function.
+Another feature of `AsyncInferQueue` is ability to setting callbacks. When callback is set, then any job that ends inference calls upon Python function. Callback function must have two arguments. First is the request that calls the callback, it provides `InferRequest` API. Second one, being called "userdata", provides possibility of passing runtime values. Those values can be of any Python type and later used inside callback function.
 
 The callback of `AsyncInferQueue` is uniform for every job. When executed, GIL is acquired to ensure safety of data manipulation inside the function.
 
 @snippet docs/snippets/ov_python_exclusives.py asyncinferqueue_set_callback
 
-### Working with u1, u4 and i4 element types
+### Working with u1, u4 and i4 Element Types
 
-Since openvino supports low precision element types there are few ways how to handle them in python.
-To create an input tensor with such element types you may need to pack your data in new numpy array which byte size matches original input size:
+Since OpenVINO™ supports low precision element types, there are few ways to handle them in Python.
+To create an input tensor with such element types, you may need to pack your data in the new *numpy* array, which byte size matches original input size:
 @snippet docs/snippets/ov_python_exclusives.py packing_data
 
-To extract low precision values from tensor into numpy array you can use next helper:
+To extract low precision values from a tensor into the *numpy* array you can use next helper:
 @snippet docs/snippets/ov_python_exclusives.py unpacking
 
 ### Releasing the GIL
 
-Some functions in Python API release the Global Lock Interpreter (GIL) while running work-intensive code. It can help you to achieve more parallelism in your application using Python threads. For more information about GIL please refer to the Python documentation.
+Some functions in Python API release the Global Lock Interpreter (GIL) while running work-intensive code. It can help you achieve more parallelism in your application, using Python threads. For more information about GIL, refer to the Python documentation.
```

Comment:
```suggestion
Some functions in Python API release the Global Lock Interpreter (GIL) while running work-intensive code. This can help you achieve more parallelism in your application, using Python threads. For more information about GIL, refer to the Python documentation.
```